src/convertToObject.js

```diff
@@ -5,8 +5,26 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
+function convertToObject(stylesString) {
+  return stylesString
+    .split(';')
+    .map((style) => style.trim())
+    .filter(Boolean)
+    .map((style) => {
+      const [property, value] = style.split(':');
+
+      if (property === undefined || value === undefined) {
+        return null;
+      }
+
+      return [property.trim(), value.trim()];
+    })
+    .filter(Boolean)
+    .reduce((obj, [property, value]) => {
+      obj[property] = value;
+
+      return obj;
+    }, {});
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have the same indentation to improve readability. Align the method calls so that the dots leading the method calls are vertically aligned.